db/migrate/20250325114517_remove_enabled_from_authentication_tokens.rb

```diff
@@ -0,0 +1,7 @@
+# frozen_string_literal: true
+
+class RemoveEnabledFromAuthenticationTokens < ActiveRecord::Migration[7.2]
+  def change
+    safety_assured { remove_column :authentication_tokens, :enabled, type: :boolean, default: true, null: false }
```

Comment:
I suggest to check if any existing token has `enabled: false` on `sandbox` in case we need to change its state.